sfepy/examples/acoustics/vibro_acoustic3d_mid.py

```diff
@@ -62,15 +62,15 @@
 wc2 = w * c2
 
 regions = {
-    'Gamma0': 'all',
+    'Gamma0_1': 'all',
```

Comment:
Why this renaming? Also the docstrings use Gamma0.

Reply:
The same region is named `Gamma0_1` in `vibro_acoustic3d.py`.  `vibro_acoustic3d_mid.py` is subproblem of   `vibro_acoustic3d.py` and the results are merged into the one output file.

Reply:
OK, maybe mention it in the file?